### DISCUSSION TITLE
Did anybody try Gaphite with Ubuntu 22.04 ?

### DISCUSSION THREAD

**Discussion by 3Paul** [graphite-project/graphite-web#2786] [Q&A]
Hello.
Could you please help. 
I found that Gaphite presents in Ubuntu 22.04 repository but it does not work with provided settings/scripts.
Did anybody try it with Ubuntu 22.04? Maybe available steps how to install it and fix for Ubuntu 22?

**Comment by bucko909** [comment]
You'll probably want to report this downstream to Ubuntu; I'm on Debian 12 and it seems to work fine there, so there shouldn't be anything preventing Ubuntu from having a working package.
